lms/static/js/fixtures/discovery.html

```diff
@@ -1,26 +1,28 @@
 <section class="courses-container">
-  <div id="discovery-form">
-    <form>
+  <div id="discovery-form" class="wrapper-search-context">
+    <div id="discovery-message" class="search-status-label"></div>
+    <form class="wrapper-search-input">
       <input class="discovery-input" placeholder="Search for a course" type="text"/><!-- removes spacing
       --><button type="submit" class="button postfix discovery-submit" aria-label="Search">
         <i class="icon fa fa-search" aria-hidden="true"></i>
+        <div aria-live="polite" aria-relevant="all">
```

Comment:
This is file actually just the fixture for js tests, which should mimic lms/templates/courseware/courses.html .